Dockerfile

```diff
@@ -0,0 +1,116 @@
+# Copyright (C) 2020 Bosch Software Innovations GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+FROM frolvlad/alpine-java:jdk8-slim AS build
+
+COPY . /usr/local/src/ort
+
+WORKDIR /usr/local/src/ort
+
+RUN apk add --no-cache \
+    # Required for Node.js to build the reporter-web-app.
+    libstdc++ \
+    # Required to allow to download via a proxy with a self-signed certificate.
+    ca-certificates coreutils openssl && \
+    scripts/import_proxy_certs.sh
+
+RUN scripts/set_gradle_proxy.sh && \
+    ./gradlew --no-daemon :cli:distTar
+
+FROM openjdk:11-jre-slim-sid
+
+ENV \
+    # Package manager versions.
+    BOWER_VERSION=1.8.8 \
+    BUNDLER_VERSION=1.17.3-3 \
+    CARGO_VERSION=0.40.0-3 \
+    COMPOSER_VERSION=1.9.1-1 \
+    CONAN_VERSION=1.18.0 \
+    FLUTTER_VERSION=v1.7.8+hotfix.3-stable \
+    GO_DEP_VERSION=0.5.4-3 \
+    GO_VERSION=1.13.4 \
+    HASKELL_STACK_VERSION=1.7.1-3 \
+    NPM_VERSION=6.13.4+ds-2 \
+    PYTHON_PIP_VERSION=18.1-5 \
+    PYTHON_PIPENV_VERSION=2018.11.26 \
+    PYTHON_VIRTUALENV_VERSION=15.1.0 \
+    SBT_VERSION=0.13.13-2 \
+    YARN_VERSION=1.17.3 \
```

Comment:
Should be `1.21.1`.

Reply:
Thanks, that also was wrongly in the next commit.